src/helper-modules/ordnance-survey/dto/ordnance-survey-auth-error-response.dto.ts

```diff
@@ -0,0 +1,8 @@
+export type OrdnanceSurveyAuthErrorResponse = {
+  fault: {
```

Comment:
Is `fault` being used as a standard variable name in MDM or can we change this to `error` to stay consistent
?

Reply:
This is error type for external Ordnance Survey error. This error is not passed to MDM users. 

API-test verifies that user gets 
```
{
      statusCode: 500,
      message: 'Internal server error',
}
```